includes/wikia/services/DataMartService.class.php

```diff
@@ -838,6 +838,32 @@ public static function getTopTagsWikisByPageviews ($tagId, $startDate, $endDate,
 		return $tagViews;
 	}
 
+	/**
+	 * Gets page views for given articles
+	 *
+	 * @param array $articlesIds
+	 * @param datetime $timeId
+	 * @return array
+	 */
+	public static function getPageViewsForArticles( Array $articlesIds, $timeId ) {
```

Comment:
While this method is not strictly conected with Insights, what do you think about passing more arguments like period and wiki id?